indico/modules/rb/client/js/common/bookings/BookingEditCalendar.module.scss

```diff
@@ -51,3 +53,7 @@
     }
   }
 }
+
+.weekday {
```

Comment:
Note: I'll get around to moving these repeated `.weekday` styles into a generic stylesheet that can imported elsewhere.